backend/src/routes/api/service/mlmd/index.ts

```diff
@@ -9,8 +9,7 @@ export default proxyService<DSPipelineKind>(
     plural: 'datasciencepipelinesapplications',
   },
   {
-    port: 8443,
-    prefix: 'ds-pipeline-md-',
+    constructUrl: (resource) => resource.status.components.mlmdProxy.url,
```

Comment:
Do you need to check status?
```suggestion
    constructUrl: (resource) => resource.status?.components.mlmdProxy.url,
```

Reply:
yeah i think so. i was wondering why TS was not yelling at me for that though

Reply:
The backend and frontend folders use different TS configs, and the backend one is much more tolerant.

Reply:
shoudl be fixed now